Documentation/contributing/testing/e2e.rst

```diff
@@ -62,6 +62,10 @@ The following example will provide the steps to run one of the tests of the
 focus ``f09-datapath-misc-2`` on Kubernetes ``1.27`` with the kernel ``net-next``
 for the commit SHA ``7b368923823e63c9824ea2b5ee4dc026bc4d5cd8``.
 
+You can also perform these steps automatically using the script
+``contrib/scripts/run-gh-ginkgo-workflow.sh`` (run this script with ``-h`` for
+indications).
```

Comment:
It might make sense to move this instruction outside of the parentheses. Also, I don't think 'indications' would translate well here in this context, however, I could be wrong.

Reply:
OK, I'll fix this. Thanks a lot!